Formula/talisman.rb

```diff
@@ -1,9 +1,10 @@
 class Talisman < Formula
   desc "Tool to detect and prevent secrets from getting checked in"
   homepage "https://thoughtworks.github.io/talisman/"
-  url "https://github.com/thoughtworks/talisman/archive/v1.22.0.tar.gz"
-  sha256 "3e6a1d27e0306509ba5d197dad44dc40cff5fa4d14d85ae0e4293a0d6dfb0e2f"
+  url "https://github.com/thoughtworks/talisman/archive/v1.11.0.tar.gz"
+  sha256 "95ebb3ac0215bf43d6cdf17d320e22601a3a7228d979e5a6cbaf8c4082f9ad22"
   license "MIT"
+  version_scheme 1
```

Comment:
This is at ~40 monthly installs, so I think we can omit the `version_scheme`, as it seems likely that only a handful of users upgraded to the new version.

Reply:
agree, IMO upstream should do a roll forward release.

Reply:
> This is at ~40 monthly installs, so I think we can omit the version_scheme, as it seems likely that only a handful of users upgraded to the new version.

I do not completely agree. `version_scheme` is our tool for downgrading packages, I don't see any reason why we shouldn't use it in this small case.

Anyway, I don't have a strong opinion here, feel free to delete `version_scheme` if you think it will be better.